types/lyricist/index.d.ts

```diff
@@ -0,0 +1,333 @@
+// Type definitions for lyricist 2.2.2
+// Project: https://github.com/scf4/lyricist, https://www.npmjs.com/package/lyricist
+// Definitions by: Aleksei Klimenko <https://github.com/DadUndead>
+// Definitions: https://github.com/DefinitelyTyped/
+// TypeScript Version: 2.7
+
+declare module 'lyricist';
+
+export default class Lyricist {
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default